tests/neteventebpfext/netevent_sim/netevent_types.h

```diff
@@ -38,18 +31,56 @@ typedef enum _drop_reason
     DROP_REASON_BANDWIDTH_LIMIT = 3,
     DROP_REASON_INACTIVE_TIMEOUT = 4,
 } drop_reason;
-typedef struct _netevent_message
+typedef struct _netevent_payload
 {
-    event_header_t header;
-
+    unsigned char event_id;
     ip_address_t source_ip;
     ip_address_t destination_ip;
     unsigned short source_port;
     unsigned short destination_port;
-    unsigned short reason;
 
     // Event counter, for testing purposes
     unsigned long event_counter;
+} netevent_payload_t;
+
+// Packet descriptor used for event streaming.
+typedef struct _netevent_message_descriptor
```

Comment:
_netevent_message_descriptor, _netevent_message_metadata, _netevent_message_header are all defined in the WDK. Shouldn't we use the WDK definitions instead?